client/src/components/EditProfile/ProfilePhoto/UploadPhoto.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ChangeEvent, useState } from 'react';
+import { Avatar, Box, Typography, Button } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+import useStyles from './useStyles';
+
+const UploadPhoto: React.FC = (): JSX.Element => {
+  const classes = useStyles();
+
+  const [selectedImage, setSelectedImage] = useState<File>();
+
+  const imageChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files.length > 0) {
+      setSelectedImage(e.target.files[0]);
+    }
+  };
+
+  const avatarSrc = selectedImage && URL.createObjectURL(selectedImage);
+
+  return (
+    <Box className={classes.root} component="form">
```

Comment:
The code from this PR is already a part of our remote main, maybe consider updating this branch with the new main branch 👍🏽